src/components/Dashboard/Inspector/Inspector.jsx

```diff
@@ -42,6 +44,7 @@ export default class Inspector extends Component {
     const { widget } = this.props;
     if (widget != null && widget.device != null) {
       this.fetchAttributes(widget.device);
+      this.fetchCommands(widget.device);
```

Comment:
This and the line above are duplicated on lines 59 and 60. They could be moved to a function.